src/app/_components/DistrictSearch/List.js

```diff
@@ -0,0 +1,53 @@
+'use client';
+
+import { Tick } from '@icons/index';
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+import { useState } from 'react';
+import { SwiperSlide } from 'swiper/react';
+import { PillButton, Slider } from '@components';
+import { clsx } from 'clsx/lite';
```

Comment:
🟢 isn't there should be 'cn' ?

Reply:
Nope, it shouldn't. Cn combines classes and clsx create a string based on the conditions.